protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/PipetteFields/LabwareField.tsx

```diff
@@ -14,10 +14,18 @@ export function LabwareField(props: FieldProps): JSX.Element {
   const disposalOptions = useSelector(getDisposalOptions)
   const options = useSelector(getLabwareOptions)
   const dispatch = useDispatch()
+
+  const optionsWithoutOffDeck = options.filter(
+    option => !option.name.toLowerCase().includes('off-deck')
+  )
```

Comment:
I think it may be better to check the labware's actual location value. something like 
```suggestion
  const robotState = useSelector(getRobotStateAtActiveItem)
  const optionsWithoutOffDeck = allOptionsUnfiltered.filter(
    ({ value }) => robotState?.labware[value]?.slot !== OFFDECK
  )
```